react/features/dynamic-branding/actionTypes.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Action used to signal the branding elements are ready to be displayed
+ */
+export const SET_CUSTOMIZATION_READY = 'SET_CUSTOMIZATION_READY';
```

Comment:
lets rename these actions to use "dynamic branding" in the name:

SET_DYNAMIC_BRANDING_READY and SET_DYNAMIC_BRANDING_DATA

Also adjust the action  names accordingly please.

Reply:
Done